bin/email_notifications.js

```diff
@@ -31,7 +31,7 @@ var deliveryQueue = new SQSReceiver(config.emailNotifications.region, [
 ])
 
 DB.connect(config[config.db.backend])
-  .done(
+  .then(
```

Comment:
That's literally the first bullet point in the PR description! 😆

But, to repeat myself:

> * Unhandled rejections that reach `done` kill the process. To that end, I've replaced calls to `done` with `then`.

So it's a precautionary move really. Happy to change it back if you think I was wrong?